lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -25,18 +25,27 @@ class AppHostInstaller
           #
           attr_reader :name
 
+          # @return [PBXGroup] the group that this app host file belong in.
+          #
+          attr_reader :group
+
           # Initialize a new instance
           #
           # @param [Sandbox] sandbox @see #sandbox
           # @param [Pod::Project] project @see #project
           # @param [Platform] platform @see #platform
           # @param [String] name @see #name
+          # @param [PBXGroup] group @see #group
           #
-          def initialize(sandbox, project, platform, name)
+          def initialize(sandbox, project, platform, name, target_label)
```

Comment:
this is passed in as `target_label` but it does not match the rubydoc above.

What is the difference between `name` and `target_label`?